Source/WebCore/style/StyleAdjuster.cpp

```diff
@@ -144,7 +144,10 @@ static DisplayType equivalentBlockDisplay(const RenderStyle& style)
         return DisplayType::Grid;
     case DisplayType::Ruby:
         return DisplayType::RubyBlock;
-
+    case DisplayType::Math:
+       return DisplayType::Inline;
```

Comment:
this is incorrect, the function `equivalentBlockDisplay` is expected to return a block, so probably it should be `DisplayType::MathBlock`.